### ISSUE TITLE
Binary release

### ISSUE THREAD

**Issue by hermanzdosilovic** [sharkdp/insect#129]
Instead of installing it with `npm` i would like to see just a binary release for various platforms which I can just download and place on my computer.

Maybe [nexe](https://github.com/nexe/nexe) can help you with this.

**Comment by sharkdp** [comment]
Sounds like a good idea!

I don't have any experience with *nexe*, so any help would be appreciated.

**Comment by hermanzdosilovic** [comment]
Me neither, but I just found it and it looks easy and useful. And I don't have node on my computer to try it out.

**Comment by colinwahl** [comment]
Hey! First off, I want to say this project is really cool. Secondly, I will like to help out on this issue! To start, which platforms would we want to target?

**Comment by sharkdp** [comment]
@colinwahl Thank you very much for the feedback! I'm glad you like it.

In principle, we can target every platform that supports nodejs. However, since it's rather easy to install Insect on Linux(?) and macOS, I guess we would mainly target Windows with this "binary" release.

**Comment by colinwahl** [comment]
@sharkdp That sounds good.  I am attempting it now, but I am having a bit of trouble running the nexe compiler.  I get an error to the tune of: 
```
user@user:~/purescript/insect$ nexe index.js 
ℹ nexe 2.0.0-rc.11
Acorn error: Octal literal in strict mode (91:34)
File: insect/index.js

89              process.exit(0);
90            } else if (res.msgType == "clear") {
91              process.stdout.write('\033[2J\033[0f');
92            } else {
93              console.log(res.msg + "\n");
94            }
95          }
```

I have tried escaping the octals by changing the line 91 above to `process.stdout.write('\\033[2J\\033[0f');
`, and it compiles and runs on Windows + Mac + Linux, but the clear function doesn't work.  Do you have any experience with something like this?

**Comment by colinwahl** [comment]
After some more poking around, I read that replacing line 91 to 
```
process.stdout.write('\x1Bc');
```
doesn't break nexe anymore, and seems to have the same behavior on Windows + Linux + Mac, but I am not sure if this is the correct way to fix it.

**Comment by sharkdp** [comment]
Thank you very much for looking into this.

I'm not sure anymore where I got this clear-sequence from. I believe I never tested this on Windows. Maybe we should replace it by a function call to some npm package that provides this functionality? I'm not sure.

However, if you found a solution that works platform-independently, we could certainly change that.

I don't have enough time to look into this at the moment, but I can come back to this in a few days. Thanks!

**Comment by colinwahl** [comment]
Hey, so I did more testing and indeed the change I suggested appears to work on Windows, Linux (Ubuntu) and Mac. I also tested it on web and it has the same behavior in the jQuery console.  I could make a PR with the change, if that is alright. Then we can get to figuring out the best way to publish the binary releases.

**Comment by sharkdp** [comment]
That would be great!

**Comment by colinwahl** [comment]
Great, looks like we are ready to build the binaries.  I can write a script that does it for us for all target distributions.  Where should I put this script?  Also, do we want to distribute the binaries via adding them to releases?

**Comment by sharkdp** [comment]
> Where should I put this script?

Maybe we could add this to the `build.sh` script in the project root? Ideally, the build of *nexe* packages would be optional and could be enabled via `./build.sh --nexe` or similar.

> Also, do we want to distribute the binaries via adding them to releases?

Yes, definitely. We can either do this manually or we could even try to do this in a Travis deploy stage.

**Comment by sharkdp** [comment]
This has been fixed via #135, thanks @colinwahl.

Binaries for insect 5.0 are now available via the [release page](https://github.com/sharkdp/insect/releases/tag/v5.0.0).

/CC @hermanzdosilovic
